components/count-badge/count-badge-icon.js

```diff
@@ -79,18 +91,31 @@ class CountBadgeIcon extends CountBadgeMixin(LitElement) {
 	}
 
 	render() {
-		const numberPadding = this.size === 'small' ? '0.55rem' : '0.7rem';
+		const numberPadding = this.size === 'small' ? '-0.65rem' : '-0.7rem';
```

Comment:
So now that things are positioned absolutely, could we switch to using `top`/`right` to position the number and maybe a transform to slide it up 50% of its height?

Something like this:
https://codepen.io/dlockhart/pen/VwzdxMK

![Screen Shot 2021-11-09 at 4 38 47 PM](https://user-images.githubusercontent.com/5491151/141009241-69f12de5-a11d-4cab-9ca5-2e0cfff6daf1.png)

Reply:
Yep that seemed to work! Didn't need the wrapper div either.